.github/workflows/ci.yml

```diff
@@ -35,25 +35,25 @@ jobs:
         # If you update this list of supported compiler versions,
         # make sure to update the `tested-with` section of `universum.cabal`.
         ghc:
-          - "8.6.5"
           - "8.8.4"
           - "8.10.7"
           - "9.0.1"
           - "9.2.2"
+          - "9.4.5"
         # Use only the latest compiler on non-Linux
         exclude:
+          - os: macOS-latest
+            ghc: 9.2.2
           - os: macOS-latest
             ghc: 9.0.1
           - os: macOS-latest
             ghc: 8.8.4
-          - os: macOS-latest
-            ghc: 8.6.5
+          - os: windows-latest
+            ghc: 9.2.2
           - os: windows-latest
             ghc: 9.0.1
           - os: windows-latest
             ghc: 8.8.4
-          - os: windows-latest
-            ghc: 8.6.5
```

Comment:
:thinking: why is 8.10.7 not excluded btw?

Reply:
Looks like some old artifact, since you paid attention to this I'll fix.

Reply:
Since I'm pushing to this branch anyway, I've pushed 93da091 to fix this.

Reply:
Kaef :+1: